package protoFile.fate.YiJianHeCheng3606;
message YiJianHeChengRequest{
    required int32 id=1;//½ÇÉ«id
}
message YiJianHeChengResponse{
    optional string message=1;
    optional bool result=2;
}